src/utils.py

```diff
@@ -275,3 +297,103 @@ def upload_experiment_to_s3(experiment_id: str,
     for upload_file in tqdm(upload_files):
         upload_file = str(upload_file)
         s3_client.upload_file(upload_file, bucket_name, upload_file.replace(current_global, object_global))
+
+
+def raven_to_csv_annotations(annotations_path: str,
+                             recording_path: str,
+                             dataset_name: str = 'recordings_2018_filtered'):
+
+    """
+    Takes annotation files (selection table) created in Raven and turns it to a compatible annotations csv.
+    """
+    #create dataframe
+    annotations=Path(annotations_path)
+    recording=Path(recording_path)
+    filelist = list(annotations.glob('*selections.txt')) #ignore irrelevent files
+    metadata = []
+    for file in filelist:
+        dfTemp = pd.read_csv(file, sep="\t")
+        dfTemp['filename'] = re.search(r'18\d{4,4}_\d{6,6}', file.as_posix()).group()
+        dfTemp['StartMicInWater'] = np.amin(dfTemp['Begin Time (s)'])
```

Comment:
Also this naming. Other datasets won't have mic going in and out of water so perhaps rename?

Reply:
This seems less relevant since it doesn't affect the output. I removed all the actual code regarding out-of-water sections

Reply:
so remove this field